### PR TITLE
ship to prod

### PR DESCRIPTION
**Pull request by ericr6**
update glossary

### CODE REVIEW

**Review by PierreJeanjacquot — APPROVED** [review]
LGTM

File: for-developers/confidential-computing/app-developer-secret.md [hunk 1]
```diff
@@ -0,0 +1,367 @@
+# Use a developer secret
+
+{% hint style="success" %}
+**Prerequisites**
+
+* [Docker](https://docs.docker.com/install/) 17.05 or higher on the daemon and client.
+* [Nodejs](https://nodejs.org) 14.0.0 or higher.
+* [iExec SDK](https://www.npmjs.com/package/iexec) 7.2.0 or higher.
+* Familiarity with the basic concepts of [Intel® SGX](intel-sgx-technology.md#intel-r-software-guard-extension-intel-r-sgx) and [SCONE](intel-sgx-technology.md#scone-framework) framework.
```
Comment (jbern0rd) [inline for-developers/confidential-computing/app-developer-secret.md]: Seems to be
```suggestion
* Familiarity with the basic concepts of [Intel® SGX](intel-sgx-technology.md#intel-software-guard-extension-intel-sgx) and [SCONE](intel-sgx-technology.md#scone-framework) framework.
```

Comment (jbern0rd) [reply for-developers/confidential-computing/app-developer-secret.md]: The second syntax works for me, not the first

**Comment by izimio** [comment]
[Quick start for workers](https://documentations.iex.ec/staging/v/v7.1-update/for-workers/quick-worker-start)

Small typo while editing the title in bold 
'_'

image

**Comment by izimio** [comment]
Dead links: 
* Juste bellow [Virtual machine part](https://documentations.iex.ec/staging/v/v7.1-update/for-workers/quick-worker-start#virtual-machine-vm) =>  [dead link](https://download.iex.ec/iExec-Public-Worker.ova)
* Juste bellow [Doracl page](https://documentations.iex.ec/staging/v/v7.1-update/use-cases/iexec-doracle#the-doracle-generic-contract) => [Link to deprecated package of npm](https://www.npmjs.com/package/iexec-doracle-base )
* At the very end of the page [Doracle page](https://documentations.iex.ec/staging/v/v7.1-update/use-cases/iexec-doracle) =>  [Next / Previous link leading to nowhere](https://docs.iex.ec/consortiumdeployment.html)
* Juste bellow [PoCo]( https://documentations.iex.ec/staging/v/v7.1-update/key-concepts/proof-of-contribution#orders-description) => [All "see tag" links of the part are leading to none existing page](https://docs.iex.ec/poco.html#tag)
* Juste bellow [PoCo callback](https://documentations.iex.ec/staging/v/v7.1-update/key-concepts/proof-of-contribution#callback) => [EIP1154 and * dead links](https://docs.iex.ec/pocosrc/poco-else.html#eip1154)
* All links of the page [Migration](https://documentations.iex.ec/staging/v/v7.1-update/help/migration)
* In [Contact us](https://documentations.iex.ec/staging/v/v7.1-update/help/contact-us) => [SDK link leading to wrong place](https://documentations.iex.ec/staging/v/v7.1-update/help/contact-us)

File: for-developers/your-first-app.md [hunk 2]
```diff
@@ -47,89 +46,207 @@ Today you can run any application as a task. This means services are not support
...
+  "datasetmaxprice": "0"
+  ...
+}
+```
+
+{% endcode %}
+
+##### Consuming a dataset
+
+The **dataset** file is downloaded and unencrypted in the `IEXEC_IN` directory and gets its name exposed to the application via `IEXEC_DATASET_FILENAME`.
```
Comment (ericr6) [inline for-developers/your-first-app.md]: unencrypted => unencrypted on the fly

**Comment by PierreJeanjacquot** [comment]
@ericr6 suggestion added e15993e26543780527e34c3101a02fd015e5e832

File: for-developers/your-first-app.md (hunk 2 above)
Comment (PierreJeanjacquot) [reply for-developers/your-first-app.md]: suggestion added e15993e26543780527e34c3101a02fd015e5e832

**Comment by PierreJeanjacquot** [comment]
@izimio wooo nice catch
All these points are not in the scope of the current PR, I fixed some of them, for the others I'm not sure the content is up to date, I suggest opening an issue in this repo until we have a competent volunteer to deal with them.

Fixed:
- All links of the page [Migration](https://documentations.iex.ec/staging/v/v7.1-update/help/migration) ee1302091d3a9dfee4a23f9f88b3f5df67730d84
- In [Contact us](https://documentations.iex.ec/staging/v/v7.1-update/help/contact-us) => [SDK link leading to wrong place](https://documentations.iex.ec/staging/v/v7.1-update/help/contact-us) 55c5f806c39960b4d725cd3e1d59cd711e6509a4

Not fixed:
- [For workers > Quick start](https://documentations.iex.ec/staging/for-workers/quick-worker-start) @ericr6 public workers are still not available, should we keep this section?
- [Use cases > iExec DOracle](https://documentations.iex.ec/staging/use-cases/iexec-doracle) @ericr6 any idea who can maintain this page?
- [Key concepts > Proof of contribution](https://documentations.iex.ec/staging/key-concepts/proof-of-contribution) same question

File: help/glossary.md [hunk 3]
```diff
@@ -1,41 +1,236 @@
...
+
+A standard for fungible tokens. This property makes each token exactly the same in type and value as another token.
+
+## ERC-721
+
+A standard for non-fungible tokens. This property makes each token unique and capable of having a different value from another token from the same Smart Contract. This uniqueness may come from a variety of sources, including age, rarity, or appearance.
+
+## iExec Academy
+
+Exec’s content aggregator where people can find content related to the project, including articles, demos, documentation, and tutorials.
```
Comment (jbern0rd) [inline help/glossary.md]: typo and is it at the right place in the glossary ?

File: help/glossary.md [hunk 4]
```diff
@@ -1,41 +1,236 @@
...
+
+Is a confidential computing technology developed by Intel.
+
+See “Confidential Computing” for more information.
+
+## Sidechain
+
+A controlled blockchain deployed over a data center and linked to Ethereum Mainnet with a bridge permitting to transfer assets between the two. iExec’s sidechain “Bellecour” is iExec’s mainnet bridged to ethereum mainnet.
+
+## sRLC
```
Comment (jbern0rd) [inline help/glossary.md]: Shouldn't it be between **SMS** and **Staking** ?

File: help/glossary.md (hunk 3 above)
Comment (jbern0rd) [reply help/glossary.md]: The typo is on iExec, written iExec here

Comment (PierreJeanjacquot) [reply help/glossary.md]:
```suggestion
iExec’s content aggregator where people can find content related to the project, including articles, demos, documentation, and tutorials.
```

File: help/glossary.md (hunk 4 above)
Comment (PierreJeanjacquot) [reply help/glossary.md]: yes fixed 767b6bcaaff866d988a6317d68630c9b589d5412

File: for-developers/confidential-computing/requester-secrets.md [hunk 5]
```diff
@@ -0,0 +1,399 @@
+# Use requester secrets
+
+{% hint style="success" %}
+**Prerequisites**
+
+* [Docker](https://docs.docker.com/install/) 17.05 or higher on the daemon and client.
+* [Nodejs](https://nodejs.org) 14.0.0 or higher.
+* [iExec SDK](https://www.npmjs.com/package/iexec) 7.2.0 or higher.
+* Familiarity with the basic concepts of [Intel® SGX](intel-sgx-technology.md#intel-r-software-guard-extension-intel-r-sgx) and [SCONE](intel-sgx-technology.md#scone-framework) framework.
```
Comment (jbern0rd) [inline for-developers/confidential-computing/requester-secrets.md:9]: I knew I should have told I saw it in different places :-)
```suggestion
* Familiarity with the basic concepts of [Intel® SGX](intel-sgx-technology.md#intel-software-guard-extension-intel-sgx) and [SCONE](intel-sgx-technology.md#scone-framework) framework.
```

File: for-developers/confidential-computing/create-your-first-sgx-app.md [hunk 6]
```diff
@@ -4,8 +4,8 @@
 **Prerequisites**
 
 * [Docker](https://docs.docker.com/install/) 17.05 or higher on the daemon and client.
-* [Nodejs](https://nodejs.org) 12.0.0 or higher.
-* [iExec SDK](https://www.npmjs.com/package/iexec) 7.0.0 or higher.
+* [Nodejs](https://nodejs.org) 14.0.0 or higher.
+* [iExec SDK](https://www.npmjs.com/package/iexec) 7.2.0 or higher.
 * Familiarity with the basic concepts of [Intel® SGX](intel-sgx-technology.md#intel-r-software-guard-extension-intel-r-sgx) and [SCONE](intel-sgx-technology.md#scone-framework) framework.
```
Comment (jbern0rd) [inline for-developers/confidential-computing/create-your-first-sgx-app.md:9]:
```suggestion
* Familiarity with the basic concepts of [Intel® SGX](intel-sgx-technology.md#intel-software-guard-extension-intel-sgx) and [SCONE](intel-sgx-technology.md#scone-framework) framework.
```

**Review by izimio — APPROVED** [review]

**Pull request merged**